src/docfx/build/markdown/Markup.cs

```diff
@@ -32,91 +27,101 @@ internal static class Markup
             { "Caution", "<p>Caution</p>" },
         };
 
+        private static readonly MarkdownPipeline s_markdownPipeline = CreateMarkdownPipeline();
+
+        [ThreadStatic]
+        private static (MarkupResult result, DependencyMapBuilder dependencyMap, Action<Document> buildChild) t_context;
```

Comment:
>t_context [](start = 110, length = 9)

do we have a better name? `t_context` and `Context` look confusing to me.

Reply:
`t_` is the prefix for thread static fields as described [here](https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md#c-coding-style)